steps/id-and-residency/id.tsx

```diff
@@ -126,43 +125,29 @@ const step: ProcessStepDefinition<ProcessDatabaseSchema, "id"> = {
           Component: TextAreaDetails,
           props: {
             summary: "Add note about ID if necessary" as React.ReactNode,
-            label: { value: "Notes" } as {
+            label: { value: "Notes" },
+            name: "id-notes",
+            includeCheckbox: true
+          } as {
+            summary: React.ReactNode;
+            name: string;
+            rows?: number;
+            label: {
               id?: string;
               value: React.ReactNode;
-            },
-            name: "id-notes"
+            };
+            contentBeforeTextArea?: React.ReactNode;
+            contentAfterTextArea?: React.ReactNode;
+            includeCheckbox?: boolean;
           },
```

Comment:
This type cast is getting unwieldy. Perhaps we could cast to an exported `TextAreaDetailsProps` interface instead.